### ISSUE TITLE
Set Field Array "_error" Key For A Specific Index In Field Array

### ISSUE THREAD

**Issue by asharron** [redux-form/redux-form#4260]
### Are you submitting a **bug report** or a **feature request**?
This is a feature request.

### What is the current behavior?
When performing validation upon a field array, you can set an "_error" key on that array to mark it as invalid, such as 

`errors.fieldArrayComponent._error = "Please Modify Your Submission";`

This is cool, but I validate each field in the field array and only want to mark the ones that need editing as invalid or having errors.

### What is the feature request
What I would like to be able to do is set an "_error" key on a specific index in the field array to mark just that one index as invalid. 

So basically I'm thinking of something like:

`errors.fieldArrayComponent[index] = { _error: "Please Modify This Submission" };`

That way I can show an error for just that one specific Field in the Field Array, rather than the entire array being invalid.

### Other information
I've read the documentation on the Field Array and looked through the Github issues to try and prevent this from being duplicate feature request, or something that is already implemented.

My apologizes in advance if this has already been asked. 

Please let me know if you have any questions!

**Comment by asharron** [comment]
Been so long I can't even remember what I was trying to do on this.... Closing
